### PR TITLE
Cape ratios

### PR DESCRIPTION
**Pull request by daflack**

### CODE REVIEW

File: src/CSET/operators/elevated_convection_diagnostics.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    # Filter to reduce NaN values and -inf values for plotting ease.
+    EC_Flagb[np.isnan(EC_Flagb)] = np.nan
+    EC_Flagb[np.abs(EC_Flagb) == np.inf] = np.nan
+    # Take the coordinates from an existing cube and replace the data.
+    cape_ratio_cube = SBCAPE.copy()
+    cape_ratio_cube.data = EC_Flagb
+    # Rename and remove STASH code.
+    cape_ratio_cube.var_name = "cape_ratio"
+    cape_ratio_cube.attributes.pop("STASH", None)
+    print(cape_ratio_cube)
```
Comment (jfrost-mo) [inline src/CSET/operators/elevated_convection_diagnostics.py:136]:
```suggestion
```
Unnecessary print statement.

**Review by jwarner8 — APPROVED** [review]
Science review done: all good

File: tests/operators/test_elevated_convection_diagnostics.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+"""Tests for elevated convection diagnostics."""
+
+import CSET.operators.elevated_convection_diagnostics as ecd
+import iris
```
Comment (jfrost-mo) [inline tests/operators/test_elevated_convection_diagnostics.py:6]:
```suggestion
import iris

import CSET.operators.elevated_convection_diagnostics as ecd
```
Reorganise imports.

**Review by jfrost-mo — CHANGES_REQUESTED** [review]
# Technical Review

Small changes requested.

**Review by jfrost-mo — APPROVED** [review]
Looks good to me.

**Pull request merged**